modules/debts_api/lib/debts_api/v0/vba_fsr_form.rb

```diff
@@ -4,6 +4,7 @@
 module DebtsApi
   class V0::VbaFsrForm < V0::FsrForm
     VBA_TYPE_KEY = 'DEBT'
+    VBA_AMOUNT_KEY = 'currentAR'
```

Comment:
FE has this as `currentAr` with the "r" being lowercase. Will that affect the pluck later on?

Reply:
Good catch! Fixed, and added spec to confirm.